kyuubi-server/src/main/resources/org/apache/kyuubi/ui/swagger/oauth2-redirect.html

```diff
@@ -1,3 +1,20 @@
+<!--
```

Comment:
this is not an apache file

Reply:
The original file does not have license header, we can add our license header to the original file